data/pagination.go

```diff
@@ -102,7 +102,10 @@ func getOffset(ctx context.Context, cfg *config.Config, page int, limit int) (of
 }
 
 // GetTotalPages gets the total pages of the search results
-func GetTotalPages(limit int, count int) int {
+func GetTotalPages(cfg *config.Config, limit, count int) int {
```

Comment:
Change looks fine but you should add a new test case to cover this updated behaviour.

Reply:
👍